test/e2e/gc_e2e_test.go

```diff
@@ -3,24 +3,22 @@ package e2e
 import (
 	"context"
 	"fmt"
-	"github.com/operator-framework/api/pkg/operators/v1alpha1"
-
 	"github.com/blang/semver"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/ownerutil"
+	"github.com/operator-framework/operator-lifecycle-manager/test/e2e/ctx"
+	. "github.com/operator-framework/operator-lifecycle-manager/test/e2e/dsl"
```

Comment:
OLM resources are typically imported in a third import block

Reply:
This was the result of `go fmt` but I can adjust it